src/typeahead/typeahead.ts

```diff
@@ -91,6 +91,12 @@ export class NgbTypeahead implements ControlValueAccessor,
 
 
   /**
+   * A selector specifying the element the tooltip should be appended to.
+   * Currently only supports "body".
+   */
+  @Input() container: string;
```

Comment:
> Can't we limit it to @Input() container: 'body'; in this case?

I've modeled the proposed API after all the other components that have the `container` option (tooltip, popover etc.). We can change it across the board in a separate PR.

> You should probably add it to the NgbTypeaheadConfig

Yup, good idea 👍